src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -1813,6 +1771,189 @@ public void testExecuteAndPostProcessRubricQuestion() {
         FeedbackSessionsLogic.inst().deleteFeedbackSessionCascade(fs.getFeedbackSessionName(), fs.getCourseId());
     }
 
+    @Test
+    public void testExecuteAndPostProcessRubricQuestionWeight() {
+        DataBundle dataBundle = loadDataBundle("/FeedbackSessionQuestionTypeTest.json");
+        removeAndRestoreDataBundle(dataBundle);
+
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+
+        gaeSimulation.loginAsInstructor(instructor1ofCourse1.googleId);
+
+        FeedbackSessionAttributes fs = dataBundle.feedbackSessions.get("rubricSession");
+        FeedbackQuestionAttributes fqBeforeEdit = FeedbackQuestionsLogic
+                                            .inst()
+                                            .getFeedbackQuestion(fs.getFeedbackSessionName(), fs.getCourseId(), 1);
```

Comment:
Indendation should be 8 spaces?

This might work too.
```java
FeedbackQuestionAttributes fqBeforeEdit = 
        FeedbackQuestionsLogic.inst().getFeedbackQuestion(fs.getFeedbackSessionName(), fs.getCourseId(), 1);
```